documentation/ntfs/ntfs_0.6.md

```diff
@@ -304,7 +331,8 @@ geometry_id | géometrie | Optionnel | Ce champ est un lien vers le fichier geom
 parent_station | chaine | Optionnel | Identifiant de la zone d'arrêt, utilisé que sur des arrêts de type 0 (point d'arrêt)
 stop_timezone | timezones | Optionnel | Fuseau horaire, se référer à http://en.wikipedia.org/wiki/List_of_tz_zones
 equipment_id | chaine | Optionnel | Identifiant de la propriété accessibilité
-contributor_id | chaine | Requis | Identifiant du contributeur. Cette valeur peut être vide dans le cas de la déclaration d'une zone administrative (location_type=2)
+contributor_id | chaine | Optionnel | Identifiant du contributeur. Cette valeur est nécessairement vide dans le cas de la déclaration d'une zone administrative (location_type=3)
+frame_id | chaine | Optionnel | Identifiant du jeu de données ayant fourni l'arrêt (lien vers le fichier frames). Un arrêt pouvant être partagé entre plusieurs jeux de données (d'un ou plusieurs contributeurs), le premier frame_id est retourné.
```

Comment:
I think we should write in the spec that if this field is not filled we consider the route's contributor's default frame, and if no such thing exists, we get the line's (and else the dataset default contributor"s default frame ?)